### ISSUE TITLE
ONIX techniques - 4.1 - Visual adjustments - clarity around reflowable format

### ISSUE THREAD

**Issue by chrisONIX** [w3c/publ-a11y#418]
I had some feedback about this section. 
From ONIX code list 175 we mention E201 Fixed layout and E200 Reflowable in section 4.1.2 Variables setup but we do not mention E200 in 4.1.1 Understanding the variables and so it was not clear to the user if they should take into account E200 or not? 
Do we need to add anything to the ONIX techniques.
I do not think this impacts the EPUB technique document

**Comment by clapierre** [comment]
@gregoriopellegrino

**Comment by gregoriopellegrino** [comment]
@chrisONIX I was re-thinking about this, reading through how we do the calculation... do we really need to extract the reflowable information?

In the end if the publisher declares that in the publication "All textual content can be modified" I don't think we need to do any further checking, just as we don't do it for EPUB metadata...

**Comment by chrisONIX** [comment]
@gregoriopellegrino I agree with you here. If the code 36 from list 196 is present we do not need to check for the values from list 175.
I am happy to close this one
